<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="fileTreeView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="appPathLeb">
            <property name="text">
             <string>打包程序路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="appPathLineE"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectAppPathBtn">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="installLbe">
            <property name="text">
             <string>导出安装器路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="installPathLineE"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectInstallPathBtn">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="installerNameLbe">
            <property name="text">
             <string>安装器名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="installerNameLineE"/>
          </item>
          <item>
           <widget class="QRadioButton" name="installPathQbtn">
            <property name="text">
             <string>默认</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="appIconLel">
            <property name="text">
             <string>程序图标：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="appIconPathLineE"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectAppIconBtn">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="appIconQbtn">
            <property name="text">
             <string>默认</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QPushButton" name="buildBtn">
            <property name="text">
             <string>脚本生成</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="previewBtn">
            <property name="text">
             <string>预览脚本</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="packBtn">
          <property name="text">
           <string>一键打包</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QListWidget" name="tipListW"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
